src/InputWithLabel.js

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react';
+
+const InputWithLabel = ({
+    id,
+    children,
+    type,
+    name,
+    value,
+    isFocused,
+    onChange,
+}) => {
+
+    const inputRef = React.useRef();
+
+    React.useEffect(() => {
+
+        inputRef.current.focus();
+
+    });
+
+    return (
+        <>
+            <label htmlFor={id} > {children} </label>
```

Comment:
Your label and input are not connected.

Reply:
If I add the same ID in my input will that help?

Reply:
@leipei92 correct. But be sure that you're assigning that `id` to the correct attribute.

Reply:
I have made lots of changes. May I merge the pull request now? It seems to be working as should be now.